src/components/pokemonCard/index.js

```diff
@@ -0,0 +1,45 @@
+import React, {useState, useEffect} from 'react';
+import "./PokemonCard.css"
+import axios from "axios";
+
+
+function PokemonCard({pokemon}){
+
+    const [pokemonData,setPokemonData] = useState(null)
+    useEffect( () => {
+        async function fetchPokemon() {
+            try {
+                const result = await axios.get(pokemon);
+                setPokemonData(result.data)
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+            fetchPokemon();
+
+        },[pokemon])
+
+    function capitalizeFirstLetter(string) {
```

Comment:
Deze functie kun je beter buiten (in dit geval dus boven) het `PokemonCard` component declareren. Anders wordt hij telkens opnieuw aangemaakt bij iedere re-render.